PowerShell/ScubaGear/Modules/Providers/ExportAADProvider.psm1

```diff
@@ -47,7 +47,11 @@ function Export-AADProvider {
     $ServicePlans = $SubscribedSku.ServicePlans | Where-Object -Property ProvisioningStatus -eq -Value "Success"
 
     #Obtains license information for tenant and total number of active users
-    $LicenseInfo = $SubscribedSku | Select-Object -Property Sku*, ConsumedUnits, PrepaidUnits | ConvertTo-Json -Depth 3
+    if ($null -eq $SubscribedSku)
+        {$LicenseInfo = "[]"}
+    else{
+        $LicenseInfo = $SubscribedSku | Select-Object -Property Sku*, ConsumedUnits, PrepaidUnits | ConvertTo-Json -Depth 3
+    }
```

Comment:
I didn't test this, but I think a cleaner solution would be something like:

$LicenseInfo = ConvertTo-Json -Depth 3 @($SubscribedSku | Select-Object -Property Sku*, ConsumedUnits, PrepaidUnits)

In the case where $SubscribedSku is null, this should reduce to an empty list.  In the case where its not we should extract what we want.

Can you add a comment asking us to add a test case here: https://github.com/cisagov/ScubaGear/blob/main/PowerShell/ScubaGear/Testing/Unit/PowerShell/Providers/DefenderProvider/Export-DefenderProvider.Tests.ps1

It looks like the tests need some refactoring to support testing this situation with how we're mocking stuff up, so I won't ask for that here.

Reply:
That should work. I was just copying the original logic.